source/scriptHandler.py

```diff
@@ -290,9 +290,10 @@ def executeScript(script, gesture):
 	try:
 		scriptTime = time.time()
 		scriptRef = weakref.ref(scriptFunc)
-		if (scriptTime - _lastScriptTime) * 1000 <= config.conf["keyboard"][
-			"multiPressTimeout"
-		] and scriptFunc == lastScriptRef:
+		if (
+			(scriptTime - _lastScriptTime) * 1000 <= config.conf["keyboard"]["multiPressTimeout"]
+			and scriptFunc == lastScriptRef
```

Comment:
@CyrilleB79 you asked what I meant about "preferred linting" or some such phrase I used. I meant, NVDA's preferred coding style.
Specifically, we used to have a rule, that conditionals should start new `and`/`or` clauses on new lines.

However, I checked `codingStandards.md`, and this rule is no longer extant.
Don't know why it was removed, unless it's in some other file, but it probably happened in the recent docs re-org. @seanbudd did I miss that we stopped caring about this, or is it somewhere else?